domains/your-domain-name.json

```diff
@@ -0,0 +1 @@
+centi.is-a.dev
```

Comment:
Invalid file contents, please read the docs before creating a PR: https://docs.is-a.dev